shell/renderer/api/electron_api_web_frame.cc

```diff
@@ -127,7 +128,7 @@ class ScriptExecutionCallback : public blink::WebScriptExecutionCallback {
 
   void Completed(
       const blink::WebVector<v8::Local<v8::Value>>& result) override {
-    v8::Isolate* isolate = v8::Isolate::GetCurrent();
+    v8::Isolate* isolate = JavascriptEnvironment::GetIsolate();
```

Comment:
`JavascriptEnvironment` is not initialized in the renderer

Reply:
whoops, overdid it on the batch replace